docs/machine_specific_instructions/lanl.rst

```diff
@@ -1,25 +1,25 @@
 LANL Institutional Computing
 ============================
 
-See https://int.lanl.gov/hpc/institutional-computing/index.shtml
+`LANL IC overview and search <https://int.lanl.gov/hpc/institutional-computing/index.shtml>`_
 
-DST Calendar: http://hpccalendar.lanl.gov/
+`DST Calendar <http://hpccalendar.lanl.gov/>`_
```

Comment:
This one doesn't work for me but I assume it's a proxy issue

Reply:
added note.